latest 1.17.1 version not working on server (fabric)

[main/FATAL]: Failed to start the minecraft server java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'hardcorequesting'! at Not Enough Crashes deobfuscated stack trace.(1.17.1+build.61) ~[?:?] at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke0(EntrypointUtils.java:50) ~[fabric-server-launch.jar:?] at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke(EntrypointUtils.java:33) ~[fabric-server-launch.jar:?] at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointServer.start(EntrypointServer.java:33) ~[fabric-server-launch.jar:?] at net.minecraft.server.Main.main(Main:97) [intermediary-server.jar:?] at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?] at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?] at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?] at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?] at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:234) [fabric-server-launch.jar:?] at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:153) [fabric-server-launch.jar:?] at net.fabricmc.loader.launch.knot.KnotServer.main(KnotServer.java:28) [fabric-server-launch.jar:?] at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?] at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?] at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?] at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?] at net.fabricmc.loader.launch.server.FabricServerLauncher.launch(FabricServerLauncher.java:68) [fabric-server-launch.jar:?] at net.fabricmc.loader.launch.server.FabricServerLauncher.setup(FabricServerLauncher.java:114) [fabric-server-launch.jar:?] at net.fabricmc.loader.launch.server.FabricServerLauncher.main(FabricServerLauncher.java:56) [fabric-server-launch.jar:?] Caused by: java.lang.NoClassDefFoundError: net/minecraft/class_4597 at java.lang.Class.forName0(Native Method) ~[?:?] at java.lang.Class.forName(Class.java:467) ~[?:?] at net.fabricmc.loader.util.DefaultLanguageAdapter.create(DefaultLanguageAdapter.java:50) ~[fabric-server-launch.jar:?] at net.fabricmc.loader.EntrypointStorage$NewEntry.lambda$getOrCreate$0(EntrypointStorage.java:106) ~[fabric-server-launch.jar:?] at java.util.HashMap.computeIfAbsent(HashMap.java:1219) ~[?:?] at net.fabricmc.loader.EntrypointStorage$NewEntry.getOrCreate(EntrypointStorage.java:104) ~[fabric-server-launch.jar:?] at net.fabricmc.loader.EntrypointStorage.lambda$getEntrypointContainers$1(EntrypointStorage.java:191) ~[fabric-server-launch.jar:?] at net.fabricmc.loader.entrypoint.EntrypointContainerImpl.getEntrypoint(EntrypointContainerImpl.java:37) ~[fabric-server-launch.jar:?] at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke0(EntrypointUtils.java:47) ~[fabric-server-launch.jar:?] ... 17 more Caused by: java.lang.ClassNotFoundException: net.minecraft.class_4597 at jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641) ~[?:?] at jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188) ~[?:?] at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?] at net.fabricmc.loader.launch.server.InjectingURLClassLoader.loadClass(InjectingURLClassLoader.java:57) ~[fabric-server-launch.jar:?] at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?] at net.fabricmc.loader.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:175) ~[fabric-server-launch.jar:?] at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?] at java.lang.Class.forName0(Native Method) ~[?:?] at java.lang.Class.forName(Class.java:467) ~[?:?] at net.fabricmc.loader.util.DefaultLanguageAdapter.create(DefaultLanguageAdapter.java:50) ~[fabric-server-launch.jar:?] at net.fabricmc.loader.EntrypointStorage$NewEntry.lambda$getOrCreate$0(EntrypointStorage.java:106) ~[fabric-server-launch.jar:?] at java.util.HashMap.computeIfAbsent(HashMap.java:1219) ~[?:?] at net.fabricmc.loader.EntrypointStorage$NewEntry.getOrCreate(EntrypointStorage.java:104) ~[fabric-server-launch.jar:?] at net.fabricmc.loader.EntrypointStorage.lambda$getEntrypointContainers$1(EntrypointStorage.java:191) ~[fabric-server-launch.jar:?] at net.fabricmc.loader.entrypoint.EntrypointContainerImpl.getEntrypoint(EntrypointContainerImpl.java:37) ~[fabric-server-launch.jar:?] at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke0(EntrypointUtils.java:47) ~[fabric-server-launch.jar:?] ... 17 more

--- a/fabric/src/main/java/hardcorequesting/fabric/HardcoreQuestingFabric.java
+++ b/fabric/src/main/java/hardcorequesting/fabric/HardcoreQuestingFabric.java
@@ -28,6 +28,7 @@ import hardcorequesting.common.util.Fraction;
 import hardcorequesting.fabric.capabilities.ModCapabilities;
 import hardcorequesting.fabric.tileentity.BarrelBlockEntity;
 import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
 import net.fabricmc.api.ModInitializer;
 import net.fabricmc.fabric.api.client.event.lifecycle.v1.ClientTickEvents;
 import net.fabricmc.fabric.api.client.itemgroup.FabricItemGroupBuilder;
@@ -292,6 +293,7 @@ public class HardcoreQuestingFabric implements ModInitializer, AbstractPlatform 
     }
     
     @Override
+    @Environment(EnvType.CLIENT)
     public void renderFluidStack(FluidStack fluid, PoseStack matrices, int x1, int y1, int x2, int y2) {
         // Behaves like FluidVolume.renderGuiRect(), but uses the provided PoseStack
         List<FluidRenderFace> faces = new ArrayList<>();

--- a/gradle.properties
+++ b/gradle.properties
@@ -1,6 +1,6 @@
 org.gradle.jvmargs=-Xmx4G
 
-mod_version=5.7.0
+mod_version=5.7.1
 maven_group=me.shedaniel
 archives_base_name=HQM
 

